AutoGrade/models.py

```diff
@@ -103,6 +104,8 @@ class Assignment(models.Model):
     due_date = models.DateTimeField('due date', default=datetime.now)
     publish_date = models.DateTimeField('date published', default=datetime.now)
 
+    #allowed_late_days = models.IntegerField(default=3)
```

Comment:
If this is commented out, it should preferably just be removed from the commit.

Reply:
Mistake noted